### PR TITLE
DBZ-1966 Fix NPE during quarkus:dev operation of Outbox Extension

### PR DESCRIPTION
**Pull request by Naros**
https://issues.redhat.com/browse/DBZ-1966

### CODE REVIEW

**Comment by gunnarmorling** [comment]
Hum, hum, seeing an NPE on CI in the outbox test now :(

**Comment by gunnarmorling** [comment]
Might have to try loading the resource via both ways and then raise a meaningful exception if it cannot be found?

**Comment by Naros** [comment]
Hi @gunnarmorling, that's what I ended up doing.  I honestly didn't expect fixing one to break the other mode of operation but from what I can gather this seems to work for both.

File: debezium-quarkus-outbox/runtime/src/main/java/io/debezium/outbox/quarkus/internal/AdditionalJaxbMappingProducerImpl.java [hunk 1]
```diff
@@ -66,8 +78,20 @@
                 }
             }
         }
-        catch (Exception e) {
-            throw new RuntimeException("Failed to submit OutboxEvent.hbm.xml mapping to Hibernate ORM", e);
+        catch (IOException e) {
+            throw new MappingException("Failed to read OutboxEvent.hbm.xml", e);
```
Comment (gunnarmorling) [inline debezium-quarkus-outbox/runtime/src/main/java/io/debezium/outbox/quarkus/internal/AdditionalJaxbMappingProducerImpl.java:82]: Can we use `DebeziumException` instead? Feels a bit odd to raise a Hibernate exception type here.

**Comment by gunnarmorling** [comment]
Applied; thx!

**Pull request merged**